setup.cfg

```diff
@@ -1,2 +1,2 @@
-[pycodestyle]
```

Comment:
Do we want to also support pydocstyle?

Reply:
Possibly. I don't think we have any docstrings right now anyway, and I think that could be a separate patch.